### DISCUSSION TITLE
Open new tab with bat file

### DISCUSSION THREAD

**Discussion by EtienneDuv** [cmderdev/cmder#2594] [Show and tell]
With last stable build, the script I use to open multiple tabs and setup my environment broke, so I just want to share my fix. 

Before, I would do this: 
`cmd /k  "cd D:/path/to/work/folder && ..." -new_console:t:MyNewTab`
But with last build it returned "The filename, directory name, or volume label syntax is incorrect."

To fix that, put the "new tab" command before running the cmd commands, so like this: 
`cmd /k -new_console:t:MyNewTab "cd D:/path/to/work/folder && ..."`

Hope it helps someone
Cheers

**Comment by ppatel08** [comment]
It helped me out lot. Thank you so much. I wish there was a documentation on how to do all this
